app/assets/stylesheets/components/_reorderable-list.scss

```diff
@@ -0,0 +1,128 @@
+.gem-c-reorderable-list {
+  @include govuk-font(19, bold);
+
+  list-style-type: none;
+  margin-bottom: govuk-spacing(6);
+  margin-top: 0;
+  padding-left: 0;
+  position: relative;
+
+  .govuk-form-group {
+    margin-bottom: 0;
+  }
+}
+
+.gem-c-reorderable-list__item {
+  margin-bottom: govuk-spacing(3);
+  border: 1px solid $govuk-border-colour;
+  padding: govuk-spacing(3);
+}
+
+.gem-c-reorderable-list__item--chosen {
+  background-color: govuk-colour('light-grey');
+  outline: 2px dotted $govuk-border-colour;
+
+  &:after {
+    content: none;
+  }
```

Comment:
I removed it and it seems to still behave a look the same - maybe @alex-ju would know?

Reply:
In [Content Publisher's implementation](https://github.com/alphagov/content-publisher/blob/master/app/assets/stylesheets/components/_reorderable-list.scss#L18-L27) we use the after pseudoelement to render the border (border wouldn't work because we have the marker pseudoclass used in an ordered list and doesn't cover the full width); when dragging a component (which enables the `--chosen` modifier) we didn't want that border to be visible on the 'floating' entry. Hope that helps.

Reply:
@DilwoarH it looks like we've still got this to resolve. It seems we've not ported over the code that this selector overrides: https://github.com/alphagov/content-publisher/blob/master/app/assets/stylesheets/components/_reorderable-list.scss#L18-L27

So I guess our options are to remove this selector or to establish we need the Content Publisher after code?

Reply:
I've removed it - We'll need to see how the content publisher behaves when we implement it there and add an appropriate fix.